### PR TITLE
add rule package_postfix_installed

### LINKED ISSUE #9176: Ansible for postfix_client_configure_mail_alias_postmaster fails no such file or directory
#### Description of problem:
Ansible remediation for rule `postfix_client_configure_mail_alias_postmaster` fail on RHEL-8.

#### SCAP Security Guide Version:
8c07d73

#### Operating System Version:
RHEL-8

#### Steps to Reproduce:

1. Run STIG playbook, e.g:
2. 'ansible-playbook -e ansible_python_interpreter=/usr/bin/python3 --skip-tags ensure_gpgcheck_never_disabled,configure_opensc_nss_db --private-key=/root/.ssh/ssg_id_ecdsa -i "192.168.122.138," /usr/share/scap-security-guide/ansible/rhel8-playbook-stig.yml'
3.
4.

#### Actual Results:
```

TASK [Deduplicate values from /etc/aliases] ************************************
skipping: [192.168.122.224]

TASK [Insert correct line to /etc/aliases] *************************************
changed: [192.168.122.224]

TASK [Update postfix aliases] **************************************************
fatal: [192.168.122.224]: FAILED! => {"changed": false, "cmd": "newaliases", "msg": "[Errno 2] No such file or directory: b'newaliases': b'newaliases'", "rc": 2}
```

#### Expected Results:
Task doesn't fail and playbook finishes.

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- add new templated rule
- add it to RHEL8 STIG profiles

#### Rationale:

- Fixes #9176

### CODE REVIEW

File: linux_os/guide/services/mail/package_postfix_installed/rule.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+documentation_complete: true
+
+title: 'The Postfix package is installed'
+
+description: |-
+    A mail server is required for sending emails.
+    {{{ describe_package_install(package="postfix") }}}
+
+rationale: |-
+    Emails can be used to notify designated personel about important
```
Comment (jan-cerny) [inline linux_os/guide/services/mail/package_postfix_installed/rule.yml:10]: Missing n
```suggestion
    Emails can be used to notify designated personnel about important
```

File: linux_os/guide/services/mail/package_postfix_installed/rule.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+references:
+    srg: SRG-OS-000046-GPOS-00022
+    stigid@rhel8: RHEL-08-030030
+
+ocil_clause: 'the package is not installed'
+
+ocil: '{{{ ocil_package(package="postfix") }}}'
+
+fixtext: '{{{ describe_package_install(package="postfix") }}}'
```
Comment (jan-cerny) [inline linux_os/guide/services/mail/package_postfix_installed/rule.yml:26]: I would use the  fixtext_package_installed macro for fixtext.

File: products/rhel8/profiles/stig.profile [hunk 3]
```diff
@@ -673,6 +673,7 @@ selections:
 
     # RHEL-08-030030
     - postfix_client_configure_mail_alias_postmaster
+    - package_postfix_installed
```
Comment (jan-cerny) [inline products/rhel8/profiles/stig.profile:676]: Will we have to do this change also in the RHEL 9 STIG?

**Comment by vojtapolasek** [comment]
@jan-cerny  actually yes, this change is needed in RHEL9 STIG as well. The thing is, that RHEL9 STIG uses a different rule - postfix_client_configure_mail_alias,
This rule was missing the execution of "new-aliases" command in the Ansible remediation. Therefore, the Ansible remediation was actually wrong. I fixed it in this PR and also made the rule applicable for RHEL9.

**Review by jan-cerny — APPROVED** [review]
I think that the fail of Ansible remediation is caused by the machine platform of these rules, because in the remediation.verbose.log I can see
```
TASK [Make sure that "/etc/aliases" has a defined value for root] **************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Update postfix aliases] **************************************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}
```

```
TASK [Make sure that "/etc/aliases" has a defined value for root] **************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

TASK [Update postfix aliases] **************************************************
skipping: [localhost] => {"changed": false, "skip_reason": "Conditional result was False"}

```

and when I run the tests locally on a VM back end they all pass. 
```
[jcerny@thinkpad scap-security-guide{pr/9191}]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel8 postfix_client_configure_mail_alias
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2022-07-22-1744/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_postfix_client_configure_mail_alias
INFO - Script correct.pass.sh using profile (all) OK
INFO - Script file_missing.fail.sh using profile (all) OK
INFO - Script no_root_alias.fail.sh using profile (all) OK
INFO - Script wrong.fail.sh using profile (all) OK
[jcerny@thinkpad scap-security-guide{pr/9191}]$ python3 tests/automatus.py rule --libvirt qemu:///system ssgts_rhel8 --remediate-using ansible postfix_client_configure_mail_alias
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2022-07-22-1748/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_postfix_client_configure_mail_alias
INFO - Script correct.pass.sh using profile (all) OK
INFO - Script file_missing.fail.sh using profile (all) OK
INFO - Script no_root_alias.fail.sh using profile (all) OK
INFO - Script wrong.fail.sh using profile (all) OK
```

**Pull request merged**